# Week 8: 3D Path Planning Parameters
occupancy_map_builder:
  ros__parameters:
    voxel_resolution: 0.2
    map_bounds_min: [-25.0,-10.0,-1.0]
    map_bounds_max: [25.0,10.0,5.0]

rrt_star_3d_planner:
  ros__parameters:
    start_xyz: [0.0,0.0,1.0]
    goal_xyz: [15.0,0.0,1.0]
    step_size: 0.5
    max_iterations: 5000
    goal_threshold: 1.0

trajectory_smoother:
  ros__parameters:
    smoothing_factor: 0.5
    max_velocity: 2.0
    max_acceleration: 1.0
